app/org/sagebionetworks/bridge/services/ParticipantService.java

```diff
@@ -185,6 +185,7 @@ public StudyParticipant getParticipant(Study study, Account account, boolean inc
         builder.withRoles(account.getRoles());
         builder.withId(account.getId());
         builder.withHealthCode(account.getHealthCode());
+        builder.withClientData(account.getClientData());
```

Comment:
Do we need to update ParticipantServiceTest.getStudyParticipantWithAccount()?

Reply:
Added